lib/elixir/lib/kernel.ex

```diff
@@ -835,7 +835,7 @@ defmodule Kernel do
   end
 
   @doc """
-  Rounds a number to the nearest integer.
+  Rounds a number to the nearest integer. Rounds half away from 0.
```

Comment:
```suggestion
  Rounds a number to the nearest integer half away from 0.
```

?

Reply:
How about `Rounds a number to the nearest integer using half away from 0.`?

Reply:
@TylerWitt :+1:

Reply:
Same here

Reply:
What do you think about?

>  If the number is equidistant to two integers, rounds away from zero.

Reply:
Updated. Thanks for the suggestions.